src/util/simple_timer.h

```diff
@@ -64,7 +64,22 @@
     // Finally, the ANSI C11 clock() function is used if no other timer
     // is available.
 
-#endif
+#endif // if _OPENMP
+
+typedef double simple_timer_t[2];
+
+// Determines how many milliseconds are in a second.
+#define MILLISECONDS_IN_SECOND 1000
+
+// An alias to assign for the timer.
+#define TIMER_ASSIGN(lhs, rhs) lhs[0] = rhs[0]; lhs[1] = rhs[1]
+// (Re)starts the timer.
+#define TIMER_RESTART(timer) simple_tic(timer)
+// Returns the nanoseconds (double) counted by the timer.
+#define TIMER_GET_NANOSECONDS(timer) timer[1]
```

Comment:
Do you think for completeness we should add the `TIMER_GET_SECONDS` counterpart ?